src/components/common/ColonyDecisions/DraftDecisionItem/DraftDecisionActions.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import Button from '~shared/Button';
+import { useDialog } from '~shared/Dialog';
+import { DecisionDialog, DeleteDecisionDialog } from '~common/ColonyDecisions';
+
+import { DraftDecisionItemProps } from './DraftDecisionItem';
+
+import styles from './DraftDecisionActions.css';
+
+const displayName = 'common.ColonyDecisions.DraftDecisionActions';
+
+type DraftDecisionActionsProps = DraftDecisionItemProps;
+
+const DraftDecisionActions = ({ decision }: DraftDecisionActionsProps) => {
+  const openDeleteDecisionDialog = useDialog(DeleteDecisionDialog);
+  const openDecisionDialog = useDialog(DecisionDialog);
+
+  return (
+    <div className={styles.buttonContainer}>
+      <Button
+        appearance={{ theme: 'blue' }}
+        text={{ id: 'button.delete' }}
+        onClick={(e) => {
+          e.stopPropagation();
```

Comment:
Prevents the click event on the parent from firing (i.e. the redirect to preview)